.travis.yml

```diff
@@ -1,23 +1,25 @@
 language: php
 
-php:
-  - 5.3
-  - 5.4
-  - 5.5
-  - 5.6
-  - 7.0
-  - 7.1
-  - nightly
-  - hhvm
-
 matrix:
+  fast_finish: true
   allow_failures:
     - php: nightly
+  include:
+    - php: 5.3
+    - php: 5.4
+    - php: 5.5
+    - php: 5.6
+    - php: 7.0
+    - php: 7.1
+      env:
+        - TEST_COVERAGE=true
```

Comment:
There aren't different code paths in this particular package, so this should be fine. I do work with other packages that have different code paths based on the PHP version, so it's important to me to run coverage reports under each version.

Thanks!